templates/compositions/ec-component-content-item/content-item.html.twig

```diff
@@ -0,0 +1,94 @@
+{% spaceless %}
+
+{#
+  /**
+   * @file
+   * Theme implementation for a ECL Content Item.
+   */
+  Parameters:
+    - "content_item" (associative array) (default: predefined structure): format:
+        {
+          description: '', (string) (default: ''): Description of Content Item
+          meta: [], (array) (default: []): Meta's for the Content Item
+          title: {}, (associative array) (default: {}): Predefined structure compatible with Link component. If Content Item type is a 'tile', only label property is required.
+          image: '', (string) (default: ''): Url/path to background image (non required if Content Item type is a 'tile')
+          image_position: 'after' (string) (Can be 'before' or 'after'. Is required only if Icon is specified)
```

Comment:
There is no `Icon` property as such, sure this comment is correct?

Reply:
:+1: